### DISCUSSION TITLE
How do I run unit tests in the Unified Platform Editor?

### DISCUSSION THREAD

**Discussion by cakeseller** [codesandbox/codesandbox-client#8487] [Q&A] [answered]
I see that the new editor (introduced in the second part of 2023) is missing the Tests tab. Does it mean that currently, it's not possible to run unit tests in the browser without switching back to the legacy editor?

**Comment by filipelima18** [comment, marked as answer]
Hi @cakeseller. That is correct. In the background, we are working on providing the Tests feature in our new editor. Until then, the suggested workaround is to switch to the legacy editor.

To open shared URLs in the legacy editor, you can use a `?editorMode=v1` query string in the address bar. You can also access the legacy editor by disabling 'Unified platform editor' in the Experiments tab of the User Settings.

**Reply by cakeseller** [reply]
Thank you Filipe!

**Reply by filipelima18** [reply]
Hi @cakeseller. I'm back with an update! We have released an unpolished version of the test runner in our new editor. You can open this new DevTool using the plus button or from the placeholder DevTool, as shown below.
![image (30)](https://github.com/user-attachments/assets/4969936c-aefd-4748-9376-f43fdb62eb27)
![image (29)](https://github.com/user-attachments/assets/5cef9b59-0e9f-476c-a992-9ed6fc5c4c7e)

We're still working on improving it, so we welcome any feedback you can share!

**Reply by cakeseller** [reply]
That is great news! Thank you for the update.
